docker/Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM ubuntu:14.04
 MAINTAINER svanoort <samvanoort@gmail.com>
 
+# User id of the build user
+ARG uid=4242
```

Comment:
I am trying to provide a default so existing automation will succeed to build the container. The limitation will still exist as before until updated.